scripts/utils.sh

```diff
@@ -8,25 +10,36 @@ function url_reachable() {
     return $?
 }
 
-
 function spawn_port_forwarding_command() {
-  kill_portforwarding_loop $1 $2 $3
-  run_in_background "scripts/port_forwarding_loop.sh $1 $2 $3"
+  service_name=$1
+  external_port=$2
+
+  cat << EOF > /etc/xinetd.d/${service_name}
+service ${service_name}
+{
+  flags		= IPv4
+  bind		= $(hostname -I | cut -d' ' -f1)
+  type		= UNLISTED
+  socket_type	= stream
+  protocol	= tcp
+  user		= root
+  wait		= no
+  redirect	= $(minikube ip) $(kubectl describe svc/${service_name} -n assisted-installer | grep NodePort | awk '{print $3}' | sed 's/\/.*//')
+  port		= ${external_port}
+  only_from	= 0.0.0.0/0
+  per_source	= UNLIMITED
 }
+EOF
 
-function run_in_background() {
-  bash -c "nohup $1  >/dev/null 2>&1 &"
+  systemctl reload xinetd
```

Comment:
u didnt start it so far. so I guess u should start it during the install. enable does not start the service

Reply:
I enable the service on _install_environment.sh_ with `--now` flag

```bash
--now            Start or stop unit in addition to enabling or disabling it
```

Reply:
Although I encountered a scenario when i'm running `kill_all_port_forwardings` which stops xinetd.
So just in case I added to start the service before the reload everytime.